extensions/azurecore/package.json

```diff
@@ -169,6 +169,7 @@
 		"@types/request": "^2.48.1",
 		"mocha": "^5.2.0",
 		"should": "^13.2.1",
-		"typemoq": "^2.1.0"
+		"typemoq": "^2.1.0",
+		"vscode": "1.1.26"
```

Comment:
You still shouldn't need this, like I said we are directly refering to the .d.ts in our src.

Reply:
I just saw your previous comment

Reply:
Presumably there is a way to run integration tests without the need to refer to vscode npm module, since git extension doesn't and it has colorized tests which run as integration tests, but maybe thats an issue for someone else to deal with...

Reply:
I looked at Git. What happens is extensions/vscode-colorize-tests/package.json has a VSCode dependency, and it finds all extensions with a `test/colorize-fixtures` folder and uses those in its test framework. So, even the built-in VSCode tests use this pattern.